pkgs/servers/gerbera/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv, fetchFromGitHub
+, cmake, pkg-config
+# required
+, libupnp, libuuid, pugixml, libiconv, sqlite, zlib, spdlog, fmt
+, pkgs
+# options
+, enableDuktape ? true
+, enableCurl ? true
+, enableTaglib ? true
+, enableLibmagic ? true
+, enableLibmatroska ? true
+, enableAvcodec ? false
+, enableLibexif ? true
+, enableExiv2 ? false
+, enableFFmpegThumbnailer ? false
+, enableInotifyTools ? true
+}:
+
+with stdenv.lib;
+let
+  optionOnOff = option: if option then "on" else "off";
+in stdenv.mkDerivation rec {
+  pname = "gerbera";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = pname;
```

Comment:
I prefer to set `repo` and `owner` explicitly, since they do not necessarily change with `pname`. That's mostly an opinion though and there is no established standard, so you don't have to change it. Just wanted to mention it :)